examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,125 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are stored in previous post processing pulled from an H5 file. The inverse of the T1(p) data is taken to produce the R1(p) data which is fit using the polyfit of the inverse of krho with two degrees of freedom. The R1(p) fit is then fed into the fitting routine for the enhancement data prior to normalization. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
```

Comment:
What example generates these?

Reply:
also, this line is way too long -- you need to wrap it so that it can be diffed

Reply:
currently generate_integrals.py generates the integrals however, this function returns the normalized Ep and so I can either adjust that here in this PR and update the function to also return the R1p rather than the T1p integrals or make that a separate PR-your call